arduino-adafruit.js

```diff
@@ -18,7 +18,11 @@ function initTasks(gulp, options) {
   // add adafruit specific libraries
   options.libraries.push('https://github.com/adafruit/Adafruit_WINC1500.git');
   options.libraries.push('RTCZero');
-  options.libraries.push('AzureIoTHub:0.2.0');
+  options.libraries.push('NTPClient');
+  options.libraries.push('AzureIoTHub');
+  options.libraries.push('AzureIoTUtility');
+  options.libraries.push('AzureIoTProtocol_HTTP');
```

Comment:
Because the issue of Arduino Azure IoT SDK, we will test HTTP at first, and following with SDK team for MQTT at the same time. So keep two libs at the same time to avoid unnecessary switch work in future.